### PR TITLE
pyboard changes needed for rshell

### PR DESCRIPTION
**Pull request by dhylands**
This is a small change I needed to make to pyboard.py in order to make rshell.py work.

### CODE REVIEW

File: tools/pyboard.py [hunk 1]
```diff
@@ -125,7 +127,7 @@ def eval(self, expression):
         ret = ret.strip()
         return ret
 
-    def exec(self, command):
+    def exec(self, command, data_consumer=None):
```
Comment (dpgeorge) [inline tools/pyboard.py]: Did you mean to pass data_consumer to exec_raw on the following line?

Comment (dhylands) [inline tools/pyboard.py]: Hmm. I don't use the data_consumer, so that line could remain the same as it was before.

**Comment by dhylands** [comment]
I updated the PR so that exec remains unmodified.

**Comment by dpgeorge** [comment]
Merged in 97ce883217c646a2d2bc9bf6bfce6e2eaa731dae.

**Pull request closed without merging**